src/main/java/com/yahoo/sketches/tuple/CompactSketch.java

```diff
@@ -96,15 +116,6 @@ public int getRetainedEntries() {
     return keys_ == null ? 0 : keys_.length;
   }
 
-  // Layout of first 8 bytes:
```

Comment:
What the review missed is that this code comment belongs to the toByteArray(), not to the PREAMBLE_LONGS constant.  Change rejected.